WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -33,6 +33,14 @@ class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	 */
 	const ERROR_MSG = '%s by a theme/plugin should start with the theme/plugin prefix. Found: "%s".';
 
+	/**
+	 * Minimal number of characters the prefix needs in order to be valid.
+	 *
+	 * @see Issue https://github.com/WordPress/WordPress-Coding-Standards/issues/1733
```

Comment:
As this is pointing to a URI, not a structural element, the `@link` tag would be more appropriate.
Also, the link should be first, with an optional description after it.

Refs:
* https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#56-link
* https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#512-see